lost-paws/lost-paws
# Added an add pet button component with styles

## Overview

**Issue Type**


**Description**
Created an add pet component that handles onClick, but only sends a message to the console until we add the logic. This case was just to render the component to the front end. Styles were added to the button itself as well as color and hover styles.

**Ticket Item**
Case to ticket [here](https://trello.com/c/GctRxF36/17-render-an-add-pet-button)



**Steps to Validate Feature **

1. Go to landing page
2. You should see an `Add Pet` button
3. Hover over the button to see the styling changes
4. If you also click on the button you should also see some styling happen

**Previous behavior**
This button is a fresh new component and before these changes, the button did not exist.

**Expected behavior**
All functionality should remain the same, but you should see a new button.
